src/App.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import { Home } from "./pages/Home/index";
-import { Favorites } from "./pages/Favorites/index";
-import Search from "./components/Search/index";
-import SearchPage from "./pages/SearchPage/index";
-import { Book } from "./pages/Book/index";
+import { Favorites } from "./pages/Favorites/";
+import Search from "./components/Search/";
+import SearchPage from "./pages/SearchPage/";
+import { Book } from "./pages/Book/";
```

Comment:
Why is Book named export and SearchPage not? Make everything one or the other, I would keep default exports

Reply:
OK, will do.